support-frontend/app/models/GeoData.scala

```diff
@@ -0,0 +1,19 @@
+package models
+
+import com.gu.i18n.{Country, CountryGroup}
+
+case class GeoData(countryCode: Option[String], stateCode: Option[String]) {
+  def countryGroup: Option[CountryGroup] = countryCode.flatMap(CountryGroup.byFastlyCountryCode)
+  def country: Option[Country] = countryCode.flatMap(CountryGroup.countryByCode)
+  def validatedStateNameForCountry: Option[String] = for {
```

Comment:
`validatedStateNameForCountry` isn't being used, do we need it here?

Reply:
I added this to make the API more useful. I can remove it if you think that's better.

Reply:
I've removed this.